src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -51,80 +47,18 @@ export function HistoryEventsTable() {
 
   useEffect(() => {
     async function getHistory() {
```

Comment:
Since this function is only used in one `if` and nowhere else, you don't need it to be named and you can just use its body in the `if`. Also, it doesn't need to be async because you don't `await` anything there, you use `then`.

Reply:
fixed